pkg/web/templates/accounts.html

```diff
@@ -1,7 +1,3 @@
-{{ define "styles" }}
-<link href="https://unpkg.com/slim-select@latest/dist/slimselect.css" rel="stylesheet"></link>
```

Comment:
This was moved to the base layout since we're using slim select on practically every page now.